### ISSUE TITLE
GCA tables for manuscript

### ISSUE THREAD

**Issue by laurafdeza** [laurafdeza/Dissertation#9]
Hola! Me pasó Cristina el script para sacar las tablas de los modelos GCA para el manuscrito y he podido arreglarlo para adaptarlo a mis grupos. La verdad es que era más accesible de como ella lo describía, por lo que no creo que haga falta que nos reunamos.
La única pregunta que tengo es si, en los pairwise comparisons, tengo que comparar todos los grupos con todos los grupos o si los chinos avanzados sólo comparo con, por ejemplo, los intermedios de chino y los avanzados de inglés? Al tener 5 grupos, si comparo a todos con todos, van a salir un montón de tablas.
@jvcasillas

**Comment by jvcasillas** [comment]
> Hola! Me pasó Cristina el script para sacar las tablas de los modelos GCA para el manuscrito y he podido arreglarlo para adaptarlo a mis grupos. La verdad es que era más accesible de como ella lo describía, por lo que no creo que haga falta que nos reunamos.
> La única pregunta que tengo es si, en los pairwise comparisons, tengo que comparar todos los grupos con todos los grupos o si los chinos avanzados sólo comparo con, por ejemplo, los intermedios de chino y los avanzados de inglés? Al tener 5 grupos, si comparo a todos con todos, van a salir un montón de tablas.
> @jvcasillas

Tienes que hacer las comparaciones que necesites para contestar tus preguntas de investigación. 
Me imagino que a estas alturas puedes prever mis respuestas (supongo que eso es bueno y malo). 
Lo que comentas es verdad... si haces todas las comparaciones vas a tener mogollón de tablas.

**Comment by laurafdeza** [comment]
He hecho los reports de los tres experimentos (sólo stress, comparando pitch y rhythm, y comparando con habilidades visuoespaciales). Si tienes tiempo, ¿podrías echarles un vistazo? Están [aquí](https://github.com/laurafdeza/Dissertation/tree/master/reports). Al final he comparado intermedios con intermedios y dentro de L1 para ver cómo mejoran.
Los modelos ahora están hechos con z-scores de visuospatial WM y los scores que salían de los random effects en las tasks de pitch, rhythm y el cochecito. Sobre esto me surgió otra pregunta. Si uso z-scores de verbal WM para ver que son [homogéneos](https://github.com/laurafdeza/Dissertation/blob/master/scripts/stress/analyses/00_homogeneity.R), me sale básicamente que p = 1 en el TOST. Si comparo simplemente "observation - mean(L1_observations)" no sale p=1 aunque igualmente tampoco salía que hubiese diferencias entre grupos. ¿Con los z-scores está bien aunque p = 1?

**Comment by jvcasillas** [comment]
A ver no sé si entiendo la pregunta. Has convertido WM a z-scores para cada grupo de manera independiente o con ambos grupos a la vez? Recuerda que el proceso de "estandarizar" una variable significa que su promedio es 0 y la desviación típica es 1. Si haces esto con cada group... los dos grupos tendrán un promedio de 0... por lo que obviamente no va a haber diferencias en un TOST.

**Comment by laurafdeza** [comment]
Claro, pero así no puedo comparar si son homogéneos. Lo que hice fue coger cada L1 y calcular los z-scores según el promedio de esa L1. Lo que se me ocurre es que, en lugar de z-score, calcule la media para cada L1 y luego la distancia de cada observation respecto a ese promedio. Podría usar esas distancias para calcular si son homogéneos?

**Comment by jvcasillas** [comment]
No sé si nos estamos entendiendo. Un zscore transformation no cambia la distribución. Su homogeneidad será extactamente como era antes de hacer la transformación. Estás hablando de homogeneidad y al mismo tiempo haces referencia a tost y no entiendo por qué. El tost (two one sided test) compara promedios. No te dirá si los dos grupos son homogéneos. Te dirá si los promedios de los dos grupos son equivalentes. No tiene nada que ver con su homogeneidad. Me explico? Entonces si quieres saber si los grupos son homogéneos tienes que hacer un test fe homogeneidad. Si quieres comparar promedios es otro test.

**Comment by laurafdeza** [comment]
Aparte he hecho también el Bartlett Test y salía que la varianza era igual en todos los grupos. El Bartlett Test cuenta como test de homogeneidad, no?

¡Muchas gracias por el feedback esta mañana! Ha sido muy útil, tanto el ejercicio en sí como vuestros comentarios.

**Comment by jvcasillas** [comment]
El test de bartlett es un test de la homogeneidad de la varianza (véase `?bartlett.test`).

**Comment by jvcasillas** [comment]
y no hay de que!
